README.md

```diff
@@ -52,3 +52,10 @@ No license is granted to the Wire trademark and its associated logos, all of whi
    chore(<project name>): Update translations for the new feature
    ```
 4. Once the PR is merged, create a new version by running `yarn release:patch`
+
+### Workflow: Add a new property
+
+1. Commit the new property (mentioning the related product in the commit message) to "wire-web-config-default" ([Example](https://github.com/wireapp/wire-web-config-default/commit/3cf240f47989474e5061111aaad2260e9466cdc3))
+1. Run `yarn release:patch` to create a new version ([Example](https://github.com/wireapp/wire-web-config-default/commit/dffe84e856c4a8d1e5c911caaa41012c0e02834a)). Please note that `yarn release:prepatch` is reserved for company config repos only.
+1. Open a specific company config repo (i.e. [wire-web-config-wire](https://github.com/wireapp/wire-web-config-wire)) and run `yarn sync` inside of it. Most likely you will get a merge conflict because the "version" property in "package.json" has changed. Accept the "version" coming from the "upstream" / "theirs". Afterwards do a merge commit with message "chore: Sync" ([Example](https://github.com/wireapp/wire-web-config-wire/commit/e574e9a36b97759c58f883b5eb8c4baef1c0b43b)).
```

Comment:
```suggestion
1. Open a specific company config repo (i.e. [wire-web-config-wire](https://github.com/wireapp/wire-web-config-wire)) and run `yarn sync` inside of it. Most likely you will get a merge conflict because the "version" property in "package.json" has changed. Accept the "version" coming from the "upstream" / "theirs". Afterwards do a merge commit with message "chore: Sync" ([example](https://github.com/wireapp/wire-web-config-wire/commit/e574e9a36b97759c58f883b5eb8c4baef1c0b43b)).
```